logstash-core/lib/logstash/runner.rb

```diff
@@ -19,9 +18,10 @@
 require "logstash/version"
 
 class LogStash::Runner < Clamp::StrictCommand
+  include LogStash::Util::Loggable
```

Comment:
I understand that it is not yet initialized... but the logging functionality inside is lazy, and only gets executed when calling from an instance. So it is not an issue